games-emulation/nestopia_libretro/nestopia_libretro-1.49_20180419.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A port of Nestopia, a Nintendo Entertainment System emulator to the \
+libretro API"
+DESCRIPTION="Nestopia is one of the most accurate NES emulators available \
+and is able to run 99% of commercial games close to perfection. This is the \
+libretro port of the emulator."
+HOMEPAGE="http://nestopia.sourceforge.net/"
+COPYRIGHT="2012-2018 Martin Freij, 0ldSk00l, the libretro team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="bfa119a5b18f4da8ab881f65980c60d56cbac553"
+SOURCE_URI="https://github.com/libretro/nestopia/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="af3aa63fdc24d224de21d43eb6b4c7808f6bb53d161d9b8882def3f723f94ea7"
+SOURCE_FILENAME="nestopia-libretro-${portVersion/_/-}-$srcGitRev.tar.gz"
+SOURCE_DIR="nestopia-$srcGitRev"
+ADDITIONAL_FILES="nestopia_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	nestopia_libretro$secondaryArchSuffix = $portVersion
+	addon:nestopia_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	retroarch$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion/_/-}/" \
+		$portDir/additional-files/nestopia_libretro.info.in \
+		> nestopia_libretro.info
+	cd libretro/
```

Comment:
For a `mv`, having a slash can be nice, e.g. `mv file directory/`.
But for a `cd libretro/` the slash does not bring much added value, so I'd drop it.

That said, if you wish to keep the slash in `cd libretro/` then that's OK for me.

Hmm, instead of dropping the slash, we could drop the whole `cd libretro/` and use `make`'s `-C` option:
```diff
-	cd libretro/
-	make GIT_VERSION=${srcGitRev:0:7} $jobArgs
+	make -C libretro GIT_VERSION=${srcGitRev:0:7} $jobArgs
```
What do you think? (Please feel free to choose any variant or keep the recipe as is.)